sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/storage/DatabaseManager.java

```diff
@@ -485,6 +516,7 @@ private void createTable(SQLiteDatabase db, String table, ContentValues schema)
                 sql.append("TEXT");
             }
         }
+        sql.append(uniqueConstraintQuery);
```

Comment:
We are already using StringBuilder, why combining StringBuilder and String.format?